docs/en/integrations/language-clients/nodejs.md

```diff
@@ -32,9 +32,9 @@ npm i @clickhouse/client
 
 ## Compatibility with ClickHouse
 
-| Client version | ClickHouse |
-|----------------|------------|
-| 0.0.1 - 0.0.9  | 22.8, 22.9 |
+| Client version | ClickHouse   |
+|----------------|--------------|
+| 0.0.1 - 0.0.11 | 22.8 - 22.11 |
```

Comment:
What's wrong with the current representation?

Reply:
well, we aren't running CI for 0.0.11 vs CH 22.8
and when 0.0.1 was released, there was no 22.11
but it will for sure work with these versions though.

Reply:
Then we need to split them by tested versions

Reply:
OK, let's do that as a follow-up.